### PR TITLE
Added AtmBalanceChanged, CommandExecuted and SocialTabletPlayerBanned

### PR DESCRIPTION
**Pull request by NCferrieri**

### CODE REVIEW

**Comment by NCferrieri** [comment]
yuyr

File: src/ServerConnection/SubscriptionEventMessage.ts [hunk 1]
```diff
@@ -1,6 +1,28 @@
 import type { CommonMessage } from './CommonMessage.js';
 import type { SubscriptionEvent } from './SubscriptionEvent.js';
 
+
```
Comment (mdingena) [inline src/ServerConnection/SubscriptionEventMessage.ts:4]:
```suggestion
```

File: src/ServerConnection/SubscriptionEventMessage.ts [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+    };
+    Change: number;
+    CurrentBalance: number;
+  };
+};
+
+type CommandExecutedEventMessage = CommonMessage<'Subscription'> & {
+  eventType: 'CommandExecuted';
+  data: {
+    Command: String,
```
Comment (mdingena) [inline src/ServerConnection/SubscriptionEventMessage.ts:20]:
```suggestion
    Command: string,
```

**Review by mdingena — CHANGES_REQUESTED** [review]
Nice work, thanks for submitting. Just need to fix one formatting issue and one type issue.

**Pull request merged**